src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -234,8 +312,9 @@ function handleStruct(item, [ atomics, enums, bitmaps, structs ])
 
   // Add a leading `_` before the name of struct to match what is done in the af-structs.zapt template.
   // For instance structs are declared as "typedef struct _{{asType label}}".
-  item.chipType = '_' + item.type;
-  item.isStruct = true;
+  item.chipType     = '_' + item.type;
+  item.isStruct     = true;
+  item.atomicTypeId = 0x49;
```

Comment:
0x49 feels hardcoded. Constant may be easier to read.

Reply:
I have removed it.